Makefile

```diff
@@ -42,7 +42,7 @@ git-repo-list-update-commit:
 
 git-repo-list-commit-if-changed: update-repo-list
 	@if [ -z "$$(git diff HEAD -- $(REPO_LIST_FILE))" ]; then \
-		echo "nothing to commit, working tree clean"; \
+		@echo "nothing to commit, working tree clean"; \
```

Comment:
ah sorry, actually I meant the echo further up. The echo statement here should already not get printed because its part of the `if` statement.